src/main/java/core/basesyntax/MainApp.java

```diff
@@ -1,5 +1,17 @@
 package core.basesyntax;
 
 public class MainApp {
+    public static void main(String[] args) {
 
+        Machine truck = new Truck();
+        Machine bulldozer = new Bulldozer();
+        Machine excavator = new Excavator();
+        Machine[] cars = {truck, bulldozer, excavator};
+
+        for (Machine car : cars) {
+            car.doWork();
+            car.stopWork();
+        }
```

Comment:
we may simplify some names but not in a way it changes the meaning of variable content. Excavator and bulldozer are not cars :)
```suggestion
        for (Machine machine : machines) {
            machines.doWork();
            machines.stopWork();
        }
```